src/createLoadableVisibilityComponent.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import { IntersectionObserver } from './capacities'
 
 let intersectionObserver
-let trackedElements = new Map()
+const trackedElements = new Map()
```

Comment:
Can this now be removed as it's handled inside the mocked IntersectionObserver?

Reply:
I'm not sure I follow—what does the mocked IntersectionObserver have to do with the `trackedElements` in `createLoadableVisibilityComponent`?

Reply:
Sorry I confused this with something else. Took another look and all is good 👍